core/src/main/resources/defaults/blueprints/cdp-opdb-710.bp

```diff
@@ -115,7 +115,7 @@
             "configs": [
               {
                 "name": "hbase_regionserver_java_heapsize",
-                "value": "7516192768"
+                "value": "6979321856"
```

Comment:
Yes, we are aware of this but in CM there is no way to specify these values as the percentage of host memory. In case a user changes the machine to have more memory they can also override the memory for the services. I believe this is documented and we also highlighted this for SMEs on the recent training.

Reply:
> I believe this is documented and we also highlighted this for SMEs on the recent training.

Can you doublecheck it, please? If it is not documented then please raise a jira for the doc, team.

Reply:
I couldn't find it in docs.cloudera.com so I filed DOCS-5918.

Reply:
OK